c10/core/blob.h

```diff
@@ -51,7 +49,7 @@ class CAFFE2_API Blob final : public c10::intrusive_ptr_target {
   /**
    * Returns the meta info of the blob.
    */
-  inline const TypeMeta& meta() const noexcept {
+  inline const caffe2::TypeMeta& meta() const noexcept {
```

Comment:
This looks weird

Reply:
It is weird. `TypeMeta` is part of the c10 module and should be in the c10 namespace. The reason it still lives in the `caffe2` namespace is that the registration macros are called from the `caffe2` namespace and only work if the `TypeMeta` class is there as well. We'll have to fix this at some point.